src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,49 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
```

Comment:
If we want this to be robust, we should start by checking for a `composer.lock` in the current directory and then if not found, look for one in the parent directory recursively.